### DISCUSSION TITLE
Strings ordering

### DISCUSSION THREAD

**Discussion by pewaru-333** [WeblateOrg/weblate#12510] [Q&A] [answered]
Is it possible to order strings in strings.xml (Android) files for all languages like in base language file in a pull request?

**Comment by nijel** [comment, marked as answer]
Not yet, see https://github.com/WeblateOrg/weblate/issues/11134.

For some formats, there is an add-on available to sort keys alphabetically, see https://docs.weblate.org/en/latest/admin/addons.html#customize-json-output.

**Reply by pewaru-333** [reply]
Thank you for reply. Is it going to be added soon?

**Reply by nijel** [reply]
It is currently on a backlog, as you can see on the issue. The biggest issue being that the underlying library doesn't allow us to change the ordering of the strings.
